.pre-commit-config.yaml

```diff
@@ -27,6 +27,7 @@ repos:
     rev: v0.3.1
     hooks:
       - id: absolufy-imports
+        exclude: examples/directml/llama_v2/chat_app/
```

Comment:
need to add this line into pyproject.toml,

please refer to https://github.com/pylint-dev/pylint/blob/df0800ec22c108cb1db244b5c8c4b25db79c58ba/examples/pyproject.toml#L2 and
https://github.com/microsoft/Olive/blob/main/pyproject.toml#L180